customer/src/test/java/com/boar/model/dto/AddressDTOTest.java

```diff
@@ -0,0 +1,85 @@
+package com.boar.model.dto;
+
+import org.apache.tomcat.util.bcel.Const;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+
+import java.util.Set;
+
+class AddressDTOTest {
+    private final ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+    private final Validator validator = factory.getValidator();
+
+    @Test
+    public void firstTest() {
+        AddressDTO addressDTO1 = new AddressDTO();
+        AddressDTO addressDTO2 = new AddressDTO();
+        AddressDTO addressDTO3 = new AddressDTO();
+        AddressDTO addressDTO4 = new AddressDTO();
+
+        addressDTO1.setStreet("Plac Bema");
+        addressDTO2.setStreet("Ulica Kościuszki");
+        addressDTO3.setStreet("Ulica Niepodległości");
+
+        addressDTO1.setHouseNumber("32a");
+        addressDTO2.setHouseNumber("32/33");
+        addressDTO3.setHouseNumber("1");
+
+        addressDTO1.setCity("Wrocław");
+        addressDTO2.setCity("Nowy Świat");
+        addressDTO3.setCity("Zielona Góra");
+
+        addressDTO1.setZipCode("50-001");
+        addressDTO2.setZipCode("66-131");
+        addressDTO1.setZipCode("65-999");
+
+        Set<ConstraintViolation<AddressDTO>> constraintViolation1 = validator.validate(addressDTO1);
+        Set<ConstraintViolation<AddressDTO>> constraintViolation2 = validator.validate(addressDTO2);
+        Set<ConstraintViolation<AddressDTO>> constraintViolation3 = validator.validate(addressDTO3);
+      //  Set<ConstraintViolation<AddressDTO>> constraintViolation4 =validator.validateValue(AddressDTO.class,addressDTO4.city,addressDTO4);
+
+        Assert.assertEquals(0, constraintViolation1.size());
+        Assert.assertEquals(0, constraintViolation2.size());
+        Assert.assertEquals(0, constraintViolation3.size());
+      //  Assert.assertEquals(0, constraintViolation4.size());
+   // Assert.assertEquals("may not be null", constraintViolation4.iterator().next().getMessage());
+
+
+    }
+
+    @Test
+    public void secondTest() {
```

Comment:
co robi second test? nazywaj to funkcjonalnościami, bo jak mi się wywali second test to nie będę wiedział o  co chodzi 🗡️